dubbo-remoting/dubbo-remoting-api/src/test/java/org/apache/dubbo/remoting/telnet/support/TelnetHandlerAdapterTest.java

```diff
@@ -0,0 +1,54 @@
+package org.apache.dubbo.remoting.telnet.support;

```

Comment:
Please add ASF license header, thx

Reply:
OK, I have added the ASF license header, thx!

Reply:
> Please add ASF license header, thx

OK,I have added the ASF license header, thx!